src/Kunstmaan/NodeBundle/Helper/NodeMenu.php

```diff
@@ -306,37 +219,22 @@ public function getActiveForDepth($depth)
      */
     public function getChildren(Node $node, $includeHiddenFromNav = true)
```

Comment:
Remove array_filter also in https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/7.x/src/Kunstmaan/NodeBundle/Helper/NodeMenuItem.php#L211 and use the var `$includeHiddenFromNav`